400-claims.md

```diff
@@ -48,6 +48,8 @@ While implementors are not REQUIRED to implement claims, this is the standard fo
 
 The CNAB claim is defined as a JSON document. The specification currently does not require that claims be formatted as Canonical JSON.
 
+The latest revision of a claim MUST be retained. Previous revisions of a claim MAY be retained to provide a history of actions performed on an installation.
```

Comment:
Why the restriction to only keeping the last "successful" installation revision? That is a deviation from how we currently retain the result of the last action in the current specification, ignoring this PR. 

Otherwise if I attempt an upgrade, and it fails, does that mean that we can just pretend it didn't happen and the last revision kept is still the successful installation? :-)

Otherwise I'm good with the suggested change, it's just that wording change (introducing allowing only retaining the last successful revision, as opposed to the revision) that I'm unsure about.

Reply:
I was thinking of a rollback use case; if a runtime must always keep track of the latest successful revision, then you'll always have the information you need in order to attempt an upgrade to that last known "good" state using the same bundle version + parameters.

In practice I think runtimes will choose to keep history for the last _n_ revisions anyway so I'm fine with the wording as is.